src/components/nodes/boundaryEvent/boundaryEventDropdownData.js

```diff
@@ -0,0 +1,30 @@
+const boundaryEventDropdownData = [
```

Comment:
This is good! I'd suggest pulling this change out to its own PR, because this branch might be long-lived as we wait until the backend support for conditionals is ready. Then this change could just be a single item added here in a single commit. That should help us avoid some conflicts on this branch.